src/classes/Util.ts

```diff
@@ -39,4 +39,20 @@ export class Util extends BaseClass {
 		const regex = /^[0-9a-f]{32}$/i;
 		return regex.test(uuid);
 	}
+
+	/**
+	 * Get player's name from their UUID
```

Comment:
Please also add comments for the uuid parameter.

Reply:
fixed in 3dabc588